applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking.yaml

```diff
@@ -19,7 +19,6 @@ extensions:
   - libgxf_cuda.so
   - libgxf_multimedia.so
   - libgxf_serialization.so
-  - libgxf_stream_playback.so
```

Comment:
@sohamm17 we need to put this line back - this would be addressed in another MR.
Note that it should work for the 1.0 release

Reply:
Added back.